src/server/game/Entities/Player/Player.cpp

```diff
@@ -2168,6 +2170,15 @@ void Player::RemoveFromWorld()
             SetViewpoint(viewpoint, false);
         }
     }
+    // TODO: FIXME
+    if (sIRC->ajoin == 1)
+    {
+        QueryResult result = WorldDatabase.PQuery("SELECT `name` FROM `irc_inchan` WHERE `name` = '%s'", Unit::GetName().c_str());
```

Comment:
I think we need to ask ourselves why a temporary list of who's in the channel even gets saved to a database to start with